src/aura/BGE_BatchGiftEntryTab/BGE_BatchGiftEntryTabHelper.js

```diff
@@ -101,6 +125,43 @@
         );
     },
 
+    sortBatchData: function(component, fieldName, sortDirection) {
+        const reverse = sortDirection !== 'asc';
+        let batchData = component.get('v.batchData');
+        batchData.sort(this.sortBy(fieldName, reverse));
+        component.set('v.batchData', batchData);
+    },
+
+    /**
+     * @description: called by sortData, sorts by provided key and direction. Provided by Salesforce lightning:datatable documentation.
+     */
+    sortBy: function (field, reverse, primer) {
```

Comment:
Something I have seen done in the past, but may have drawbacks, is including a SMALL library of re-usable functions (as a static resource?) for cases like these where there are utility methods that are likely to be used in multiple components. Is that worth considering?

Reply:
This will definitely be good practice with LWC, but are headless LCs something we want to pursue? I like the code reuse, but I think they can be really difficult to read and follow.